www/app/HardwareController.js

```diff
@@ -5609,7 +5609,7 @@ define(['app'], function (app) {
 				$("#hardwarecontent #divserial").hide();
 				$("#hardwarecontent #divremote").show();
 				$("#hardwarecontent #divlogin").show();
-				$("#hardwarecontent #hardwareparamsremote #tcpport").val(9000);
+//				$("#hardwarecontent #hardwareparamsremote #tcpport").val(9000);
```

Comment:
I think this line should be

$("#hardwarecontent #divremote #tcpport").val(9000);

could you check this ?

Reply:
I double checked , unfortunately, Your line does not work.

Reply:
Strange as this is how it is used on the other location in this file. It should not crash
Your patch just does not set the value anymore...

Reply:
I will merge this patch and have a closer look

Reply:
Checked and tested, and it works like it should
https://github.com/domoticz/domoticz/commit/1c7411dc561901d9ca430254e8db4c6be731fefb